.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+| Q | A
+| --- | ---
+| Bug fix? | [yes|no]
+| New feature? | [yes|no]
+| BC breaks? | [yes|no]
+| Deprecations? | [yes|no]
+| Tests pass? | [yes|no]
```

Comment:
Again, this won't be updated so it's retrospectively useless. I suspect we simply need to remind them to add tests.

A GitHub bot that automatically checks this would be awesome...